skyvern-frontend/.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
+  "root": true,
+  "env": { "browser": true, "es2020": true },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react-hooks/recommended",
+    "prettier"
```

Comment:
Should we also use prettier -recommended?

https://www.reddit.com/r/node/comments/14jh6hz/do_you_use_eslintconfigairbnb_if_so_why_if_not_why/

Reply:
Yes, looks like that's the recommended way.